tests/components/TokenGeneratorTest.php

```diff
@@ -58,7 +62,17 @@ public function testInitSymbolsReturnsActualNumberOfSymbols():void {
          * @return void
          */
         public function testGeneratorReturnsTokenOfSuppliedLength(int $length, int $expected):void {
-            $this->assertEquals($expected, strlen($this->tokenGenerator->generateToken($length)));
+            $this->assertEquals($expected, strlen($this->tokenGenerator->generateToken($length, ["digits"])));
+        }
+
+        /**
+         * @covers ::initSymbols
+         * @covers ::generateToken
+         *
+         * @return void
+         */
+        public function testGenerateTokenReturnsOnlyDigitsOnDigitsKey():void {
+            $this->assertEquals(0, preg_match("/([a-zA-Z])+/", $this->tokenGenerator->generateToken(32, [TokenGenerator::DIGITS])));
```

Comment:
Method tests DIGITS key specifically. Might be changed to test all keys via DataProvider later.